### ISSUE TITLE
UI improvements for testing ARAX_overlay, filter, etc

### ISSUE THREAD

**Issue by dkoslicki** [RTXteam/RTX#617]
Currently, the UI doesn't display all edge information, which would be quite nice to be able to see. For examples, for this edge:
```
{
        "confidence": null,
        "defined_datetime": null,
        "edge_attributes": [
          {
            "name": "paired_concept_frequency",
            "type": "float",
            "url": "http://cohd.smart-api.info/",
            "value": 0.00020498216366468845
          }
        ],
        "evidence_type": null,
        "id": "3459699",
        "is_defined_by": "RTX",
        "negated": null,
        "provided_by": "BioLink",
        "publications": null,
        "qedge_id": "e00",
        "qualifiers": null,
        "relation": "has_phenotype",
        "source_id": "DOID:0060227",
        "target_id": "HP:0001817",
        "type": "has_phenotype",
        "weight": null
      },
```

In the UI, when you click on this edge, all you see is: 
```
DOID:0060227 has_phenotype HP:0001817
Provenance: BioLink
```
It would be nice to have the `edge_attributes` also displayed.
Also, @isbluis , if I add a value to the `weight`, will it make the edge thicker? Or is there any way to make the edge thicker based on `edge_attribute.value` or `weight` or `confidence`?

**Comment by isbluis** [comment]
Hi David. 

I will add this today.  Which branch would you like me to push this onto?
Also, do you have a good example query I can test against?

Thanks!

**Comment by dkoslicki** [comment]
Thanks @isbluis! Perhaps the `demo` branch. And then following the [directions Eric provided](https://github.com/RTXteam/RTX/wiki/Dev-info), run `python ARAX_query.py 11` after editing it to spit out the message id via `print(json.dumps(ast.literal_eval(repr(message.id)), sort_keys=True, indent=2))`

**Comment by isbluis** [comment]
Thank you @dkoslicki .   I was not able to get the system fully running (incompatible versions of Python/numpy/etc), but I did add the features and tested with some hand-crafted data.

If you have a minute, could you have a quick look and see if you like the output? 
- Edge (and Node) properties will show up when clicked on in the "Knowledge Graph" and individual result graphs
- Edge thickness is directly taken from the node.weight value when it exists; otherwise defaults to "2".  Note that these are in pixels.  I can run some kind of normalizing once we decide on a range of values to map onto.

Please unzip and replace rtx.js in your sandbox with this:

[rtx.zip](https://github.com/RTXteam/RTX/files/4223103/rtx.zip)

Thanks!

**Comment by dkoslicki** [comment]
Ah yes, we discovered that the system now needs python>=3.7 and other updated dependencies. I'll try checking out your update tomorrow morning!

**Comment by dkoslicki** [comment]
@isbluis Haven't had a chance to test this, but from previous meetings, sounds like they have been addressed. Should this issue be closed? or are we still waiting on #671, #675?

**Comment by isbluis** [comment]
Hi @dkoslicki .  Yes, it can be closed.  I am in the process of pushing this change (along with others) into demo.
